src/cpp/transport/udp/UDPServerWindows.cpp

```diff
@@ -18,14 +18,18 @@
 namespace eprosima {
 namespace uxr {
 
-UDPServer::UDPServer(uint16_t port)
+UDPServer::UDPServer(uint16_t port, uint16_t discovery_port)
     : UDPServerBase(port),
       poll_fd_{},
       buffer_{0}
-{}
+{
+    (void) discovery_port;
+}
 
-bool UDPServer::init()
+bool UDPServer::init(bool discovery_enabled)
 {
+    (void) discovery_enabled;
```

Comment:
Should we avoid windows servers if the call was used with discovery_enabled? Returning an error maybe.

Reply:
Same answer as previous